otx/mpa/modules/models/classifiers/semisl_classifier.py

```diff
@@ -81,27 +29,22 @@ def forward_train(self, imgs, **kwargs):
             raise ValueError("'gt_label' does not exist in the labeled image")
         if "extra_0" not in kwargs:
             raise ValueError("'extra_0' does not exist in the dataset")
-        target = kwargs["gt_label"]
+        target = kwargs["gt_label"].squeeze()
         unlabeled_data = kwargs["extra_0"]
-
         x = {}
         x["labeled"] = self.extract_feat(imgs)
 
-        img_uw = unlabeled_data["weak"]["img"]
+        img_uw = unlabeled_data["img_weak"]
```

Comment:
It's minor, but I think that this is an `img` and the one used for strong augmentation should be `img_strong`. Since `img_weak` is an image that eventually uses the same augmentation as the labeled image, so I think `img_strong` should be rather distinct.

Reply:
I partially agree, but to change `img_strong`, we need to change data pipeline flow from bottom. 
I distinguished weak augmented image and strong augmented image by branching out at different time.
But for `img_strong`, absolutely another implementation will be needed.

Reply:
It's very confusing.
- Labeled image -> img : weak pipeline
- Unlabeled image
  - img : strong pipeline
  - weak_img: weak pipeline
 
Isn't it a part that can be solved by changing the key in the pipeline?

Reply:
Yes, that can't be solved by only changing key name. To use `img_strong` key, time-independent image branching pipeline is needed.

Reply:
The current pipeline structure is quite inconvenient. I'll think about it some more.

Reply:
Updated. Please review again.